src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -454,7 +467,33 @@ private function checkOrderDifferenceByExtra(array $databaseOrderItems, array $b
             $backofficeOrderItemExtra = $backofficeOrderItemExtras[$extrasCounter];
             $databaseOrderItemExtra = $databaseOrderItemExtras[$extrasCounter];
             if ($databaseOrderItemExtra !== $backofficeOrderItemExtra) {
-                throw new OrderDifferenceException('Extra metadata did not match, Backoffice extras ['.implode(',', $backofficeOrderItemExtra).'] and Shop extras ['.implode(',', $databaseOrderItemExtra).']');
+                $parsedBackofficeExtra = implode(', ', array_map(
+                   static function ($value, $key) {
+                       return sprintf('%s=%s', $key, $value);
+                   },
+                    $backofficeOrderItemExtra,
+                    array_keys($backofficeOrderItemExtra)
+                ));
+
+                $parsedShopExtra = implode(', ', array_map(
+                    static function ($value, $key) {
+                        return sprintf('%s=%s', $key, $value);
+                    },
+                    $databaseOrderItemExtra,
+                    array_keys($databaseOrderItemExtra)
+                ));
+                throw new OrderDifferenceException("Extra metadata did not match, Backoffice extras \"$parsedBackofficeExtra\" and Shop extras \"$parsedShopExtra\"", $databaseOrderItemExtras, $backofficeOrderItemExtras);
```

Comment:
It would make more sense to do the serialization when you save it [database in here](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/pull/123/files#diff-1f372b218b2012a6436b45df6a8675e141e724809cd19f5195aef39a31872d72R153), is there a reason you make it before passing to the exception?

Reply:
This serialization is only used for the exception message.
Can you confirm if it no longer make sense to add this to message as we already have the table?
![image](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/assets/18332309/2d6002d2-3d45-40ad-8694-22914a2897a0)

Reply:
indeed it makes no sense to add to message

Reply:
Removed.